### PR TITLE
Navigation swiping fix

### PR DESCRIPTION
**Pull request by csduarte**
#### Summary
This PR improves navigation by increasing the distance from the edge of the screen that a swipe gesture will be detected.

@jarredwitt

### CODE REVIEW

File: app/navigation/router.js [hunk 1]
```diff
@@ -102,9 +104,14 @@ class Router extends React.Component {
...
             }
 
-            // NavigationExperimental only creates the correct panHandlers if the panHandlers prop === undefined
-            const panHandlers = navigationProps.allowSceneSwipe ? undefined : null; // eslint-disable-line
-
+            let panHandlers = null;
+            if (navigationProps.allowSceneSwipe) {
+                panHandlers = navigationPanResponder.forHorizontal({
+                    ...cardProps,
+                    gestureResponseDistance: 60, // sets the distance from the edge for swiping
```
Comment (enahum) [inline app/navigation/router.js]: What does `60` means?

Comment (jarredwitt) [inline app/navigation/router.js]: 60 pixels. Their default threshold is 30.

Comment (enahum) [inline app/navigation/router.js]: @jarredwitt I think this is still to close to the edge, and then we may want to do the same thing for the drawers?

**Comment by jarredwitt** [comment]
@enahum changes made.

**Pull request merged**